src/Sylius/Behat/Resources/config/services/pages/admin/order.xml

```diff
@@ -20,6 +20,8 @@
     </parameters>
 
     <services>
+        <defaults public="true" />
```

Comment:
Is it needed, as all (as I suppose) services here are explicitly declared as not public?

Reply:
I just added it automatically to every services file, it can be removed in the future without much effort and no BC break.